packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/NavigationItem.tsx

```diff
@@ -109,10 +124,24 @@ export const NavigationItem = ({
             className={className}
             tabIndex={0}
             $elevation={elevation}
+            $isRounded={isRounded}
         >
             <Icon name={icon} size={iconSize} color={theme.iconSubdued} pointerEvents="none" />
-            <Translation id={nameId} values={values} />
-            {itemsCount && <Count>{itemsCount}</Count>}
+            <Paragraph
+                variant="tertiary"
+                typographyStyle={typographyStyle}
+                margin={{
+                    right: spacing,
+                    left: spacing,
+                }}
```

Comment:
Please consider using `gap` instead. This `margin` breaks a navigation in the trade section.

Big white spaces:
![image](https://github.com/user-attachments/assets/540a9586-d433-4f36-9b36-e2f539be6411)

Reply:
This is the main problem for me. )

Reply:
Yes, understandable, will bring the gap back